ROADMAP.md

```diff
@@ -0,0 +1,9 @@
+# Roadmap
+
+This file provides an overview of the direction solidus is heading.
+
+## [Solidus Admin](https://github.com/orgs/solidusio/projects/12)
+
+- Introduction of hotwire
+- Filter mechanisms for lists. E.g. option types on products, sort orders by state
```

Comment:
This is not new, we already have list filters and sorting in current backend. Worth noting is maybe the introduction of Dialogs for quick edit actions right on the index screen which prevents sorting and filtering.